cartography_tools/tools/marker_settings_widget.py

```diff
@@ -36,8 +36,8 @@ def __init__(self, show_marker_count=False, show_orientation=False, show_placeme
         self.setupUi(self)
 
         self.code_combo.setEditable(True)
-        self.code_combo.setSizePolicy(QSizePolicy.MinimumExpanding, QSizePolicy.Preferred)
-        self.code_combo.setMinimumWidth(QFontMetrics(self.font()).width('X') * 40)
+        self.code_combo.setSizePolicy(QSizePolicy.Policy.Expanding, QSizePolicy.Policy.Preferred)
```

Comment:
Seems like this change was already included in https://github.com/north-road/qgis-cartography-tools/pull/4 ?

Reply:
Yes, I switched it back.